doc/dev/tests.md

```diff
@@ -43,4 +45,4 @@ cd ~/npgsql
 dotnet test ./test/Npgsql.Tests
 dotnet test ./test/Npgsql.PluginTests
 dotnet test ./test/Npgsql.Benchmarks
-```
+```
```

Comment:
We have an empty line at the end in each file, but there it was removed @roji. You was to fast to merge this.